### ISSUE TITLE
SEP-6 custom fields not passed to process_sep6_request

### ISSUE THREAD

**Issue by yuriescl** [stellar/django-polaris#345]
SEP-6 allows custom fields (defined in `/info`) to be passed to `/deposit` and `/withdraw`, but Polaris does not include those custom fields in the `params` passed to `process_sep6_request`.

**Comment by JakeUrban** [comment]
`parse_request_args()` returns the following:
```
    return {
        "asset": asset,
        "memo_type": memo_type,
        "memo": memo,
        "lang": lang,
        "type": request.GET.get("type"),
        **extract_sep9_fields(request.GET),
    }
```
This dict, along with the SEP-10 derived `account` parameter, are passed to `process_sep6_request()`:
```
    args = parse_request_args(request)
    if "error" in args:
        return args["error"]
    args["account"] = account

    transaction_id = create_transaction_id()
    transaction = Transaction(
        id=transaction_id,
        stellar_account=account,
        asset=args["asset"],
        kind=Transaction.KIND.deposit,
        status=Transaction.STATUS.pending_user_transfer_start,
        memo=args["memo"],
        memo_type=args["memo_type"] or Transaction.MEMO_TYPES.text,
        to_address=account,
        protocol=Transaction.PROTOCOL.sep6,
    )

    try:
        integration_response = rdi.process_sep6_request(args, transaction)
    except ValueError as e:
        return render_error_response(str(e))
```

Can you detail what needs to change?

**Comment by JakeUrban** [comment]
I see you're talking about the fields description just below [this section](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#for-each-withdrawal-asset-response-contains):

> The fields object allows an anchor to describe fields that are passed into /deposit and /withdraw. It can explain standard fields like dest and dest_extra for withdrawal, and it can also specify extra fields that should be passed into /deposit or /withdraw such as an email address or bank name. If a field is part of the KYC/AML flow handled by SEP-12 or the field is handled by your interactive deposit/withdrawal flow, there's no need to list it in /info. Only fields that are passed to /deposit or /withdraw need appear here.

**Comment by JakeUrban** [comment]
Polaris extracts SEP-12 parameters but does not accept non-standardized parameters, you're right. I can't think of a reason why we can't pass every unknown attribute in with `args`. We can include this change in the next minor version release.

**Comment by JakeUrban** [comment]
Just to make sure, there's no problem with Polaris' `/info` logic right? You can specify everything you need to in the response?

**Comment by JakeUrban** [comment]
I've added the `blocks-ux` label for bugs that actually stop or inhibit UX on currently-deployed instances. Since this change is internal to the anchor and doesn't prohibit the current UX it is not as urgent.

**Comment by yuriescl** [comment]
> Just to make sure, there's no problem with Polaris' `/info` logic right? You can specify everything you need to in the response?

No problem with `/info`, it's currently sufficient.

> Since this change is internal to the anchor and doesn't prohibit the current UX it is not as urgent.

For a specific asset, I need an `amount` to be passed to SEP-6, and that is high priority for my implementation.
I do see it as urgent since this issue makes Polaris non-SEP compliant.

**Comment by yuriescl** [comment]
> For a specific asset, I need an amount to be passed to SEP-6, and that is high priority for my implementation.

To be more specific, an invoice needs to be generated when `process_sep6_request` is called (user pays the invoice to make the deposit), and to generate the invoice, an `amount` is required by the external service.

**Comment by JakeUrban** [comment]
I see, ok then I can get a fix out sooner than 1.2 (1.1.4?). I don't think this is a hard change and its non-breaking so it should be fine to do now.

**Comment by yuriescl** [comment]
> I see, ok then I can get a fix out sooner than 1.2 (1.1.4?). I don't think this is a hard change and its non-breaking so it should be fine to do now.

Yes, that woud be much appreciated :)
